public/api/v1/getuserid.php

```diff
@@ -16,9 +16,17 @@
 
 =========================================================================*/
 
+$noforcelogin = 1;
 require_once(dirname(dirname(dirname(__DIR__)))."/config/config.php");
 require_once("include/common.php");
 require_once("include/pdo.php");
+include('public/login.php');
+
+// Check for authenticated user.
```

Comment:
I think I prefer something along the lines of `isset($_SESSION['cdash']) && isset($_SESSION['cdash']['loginid'])` rather than suppressing the error.